test/test_suite_test.go

```diff
@@ -223,6 +223,8 @@ var _ = BeforeAll(func() {
 				}
 			}
 		}
+		kubectl := helpers.CreateKubectl(helpers.K8s1VMName(), logger)
+		kubectl.Apply(helpers.GetFilePath("../examples/kubernetes/prometheus.yaml"))
```

Comment:
In order to capture the cilium metrics, you should either apply `prometheus.yaml` before `cilium.yaml` or force restart cilium after starting the prometheus server. I am assuming this step of applying prometheus.yaml in `BeforeAll` will be before applying the cilium ds?

Reply:
Yes, is applied before any cilium pod is installed.

Reply:
The issue at this time, is when prometheus is installed can't be in place because no CNI at all. So need to wait until Cilium is installed, that happens on the test itself.

Reply:
OK, thanks.